### DISCUSSION TITLE
ZshHub

### DISCUSSION THREAD

**Discussion by 0xMRTT** [z-shell/zi#208] [General]
I'm keep working on the idea of providing a marketplace for zsh.

I've started a django project with Django rest framework called `zsh_hub`.
It uses Primer to look like GitHub 😎.

**Comment by ss-o** [comment]
You started a few things successfully but were not very successful with implementation and maintenance. :runner: :see_no_evil: Also I am pretty sure that you will fail until the correct steps are taken before the project, but hopefully I am wrong and always wish a great success :1st_place_medal: 

:zap: :rabbit:

**Reply by 0xMRTT** [reply]
Sorry for not being active last days. I was working on a Gnome project called AdwCustomizer

**Reply by ss-o** [reply]
For me don't need to say what you did or what plan to do, trust me :octocat: :godmode: :octocat:
